core-data-map/src/androidTest/java/com/rocket/android/core/data/map/ExampleInstrumentedTest.kt

```diff
@@ -1,24 +1,22 @@
 package com.rocket.android.core.data.map
 
-import androidx.test.platform.app.InstrumentationRegistry
 import androidx.test.ext.junit.runners.AndroidJUnit4
-
+import androidx.test.platform.app.InstrumentationRegistry
+import org.junit.Assert
 import org.junit.Test
 import org.junit.runner.RunWith
 
-import org.junit.Assert.*
-
 /**
  * Instrumented test, which will execute on an Android device.
  *
  * See [testing documentation](http://d.android.com/tools/testing).
  */
 @RunWith(AndroidJUnit4::class)
 class ExampleInstrumentedTest {
+
```

Comment:
change the name of this class, it's the common ExampleTest created by default but if you are using it, better rename.

Reply:
I will delete all the tests as they are the default ones